src/main/java/seedu/address/ui/AnalyticsPanel.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.ui;
+
+import javafx.beans.property.ObjectProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.chart.PieChart;
+import javafx.scene.layout.Region;
+import seedu.address.model.person.Analytics;
+
+/**
+ * Panel containing the analytics of the loan records.
+ */
+public class AnalyticsPanel extends UiPart<Region> {
+    private static final String FXML = "AnalyticsPanel.fxml";
+
+    @FXML
+    private PieChart pieChart;
+
+    /**
+     * Creates a {@code AnalyticsPanel} with the given {@code ObjectProperty}.
+     */
+    public AnalyticsPanel(ObjectProperty<Analytics> analytics) {
+        super(FXML);
+        pieChart.setData(FXCollections.observableArrayList());
+        analytics.addListener((observable, oldValue, newValue) -> {
+            updateChart(newValue);
+        });
+    }
+
+    private void updateChart(Analytics analytics) {
+        System.out.println("Updating chart");
```

Comment:
Might it be better not to have `System.out.println` statements (instead using `Logger` or such to track the GUI)?

Reply:
Forgot to remove these, thanks for pointing this out!